python/salpytools/states.py

```diff
@@ -34,33 +34,33 @@
 state_enumeration["STANDBY"] = 1
 state_enumeration["DISABLED"] = 2
 state_enumeration["ENABLED"] = 3
-state_enumeration["FAULT"]   = 4
+state_enumeration["FAULT"] = 4
 state_enumeration["INITIAL"] = 5
-state_enumeration["FINAL"] =   6
+state_enumeration["FINAL"] = 6
 
 # This matrix expresses valid transitions and is reproduced in code afterwards.
 #
-#    \NEXT STATE
-#STATE\
-#      \ |Offline |Standby |Disabled|Enabled |Fault   |Initial |Final   |
-#------------------------------------------------------------------------ 
-#Offline | TRUE   | TRUE   |        |        |        |        |  TRUE  |
-#------------------------------------------------------------------------
-#Standby |  TRUE  | TRUE   |  TRUE  |        |  TRUE  |        |  TRUE  |
-#------------------------------------------------------------------------
-#Disable |        |  TRUE  |  TRUE  |  TRUE  |  TRUE  |        |        |
-#------------------------------------------------------------------------
-#Enable  |        |        |  TRUE  |  TRUE  |  TRUE  |        |        |
-#------------------------------------------------------------------------
-#Fault   |        |        |        |        |  TRUE  |        |        |
-#------------------------------------------------------------------------
-#Initial |        |  TRUE  |        |        |        | TRUE   |        |
-#------------------------------------------------------------------------
-#Final   |        |        |        |        |        |        | TRUE   |
-#------------------------------------------------------------------------
+#     \NEXT STATE
+# STATE\
+#       \ |Offline |Standby |Disabled|Enabled |Fault   |Initial |Final   |
+# ------------------------------------------------------------------------
+# Offline | TRUE   | TRUE   |        |        |        |        |  TRUE  |
+# ------------------------------------------------------------------------
```

Comment:
The TRUEs along the diagonal are sometimes offset by one space, but even more important, it was just decided this week that they should not be TRUE.

Reply:
This is part of another ticket DM-15818